bindings/capacitorjs/android/src/main/java/org/iota/iotawalletmobile/IotaWalletMobile.java

```diff
@@ -0,0 +1,187 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+package org.iota.iotawalletmobile;
```

Comment:
Seems more consistent with the other packages.

```suggestion
package org.iota.IotaWalletMobile;
```